blueocean-dashboard/src/main/js/components/karaoke/components/LogConsole.jsx

```diff
@@ -75,8 +75,11 @@ export class LogConsole extends Component {
     componentWillReceiveProps(nextProps) {
         // eslint-disable-line
         logger.debug('newProps isArray', Array.isArray(nextProps.logArray));
+        if (nextProps.logArray === undefined)
```

Comment:
Please add some curly braces. Details on setting up a commit-hook for prettier.js are here: https://github.com/jenkinsci/blueocean-plugin#source-code-formatting